src/middleware/middleware.js

```diff
@@ -12,8 +12,7 @@ import { useNavStore } from '~/stores/nav'
  *
  * Messages sent to the outer window have the following format:
  * {type: <event type>, value: <event value>}.
- * Currently, two event types are used:
- * - `resize` sends the height of the window (see `src/mixins/iframe-height.js`)
+ * Currently, one event types is used:
```

Comment:
```suggestion
 * Currently, one event type is used:
```

Small typo fix, now that there's only one item.